osgi.enroute.examples.webserver.application/osgi.enroute.examples.webserver.bndrun

```diff
@@ -30,6 +29,6 @@ JPM-Command:					websrvr
 	osgi.enroute.examples.webserver.application;version=snapshot,\
 	osgi.enroute.examples.webserver.webresource;version=snapshot,\
 	osgi.enroute.executor.simple.provider;version='[1.0.0,1.0.1)',\
-	osgi.enroute.logger.simple.provider;version='[1.1.1,1.1.2)',\
+	osgi.enroute.logger.simple.provider;version='[1.2.1,1.2.2)',\
 	osgi.enroute.twitter.bootstrap.webresource;version='[3.3.5,3.3.6)',\
 	osgi.enroute.web.simple.provider;version='[1.3.3,1.3.4)'
```

Comment:
It is probably better if we make this an error as well for the aforementioned reasons